e2e/playwright/list-views/src/tests/sort-list-login.spec.ts

```diff
@@ -0,0 +1,152 @@
+/*!
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { expect } from '@playwright/test';
+import {
+  ApiClientFactory,
+  FileActionsApi,
+  LoginPage,
+  NodesApi,
+  PersonalFilesPage,
+  TEST_FILES,
+  Utils,
+  test,
+  timeouts
+} from '@alfresco/playwright-shared';
+
+let initialSortState: {
+  sortingColumn: string;
+  sortingOrder: string;
+  firstElement: string;
+};
+
+async function getSortState(myPersonalFiles: PersonalFilesPage): Promise<typeof initialSortState> {
+  return {
+    sortingColumn: await myPersonalFiles.dataTable.getSortedColumnHeaderText(),
+    sortingOrder: await myPersonalFiles.dataTable.getSortingOrder(),
+    firstElement: await myPersonalFiles.dataTable.getFirstElementDetail('Name')
+  };
+}
+
+test.describe('Remember sorting', () => {
+  interface NodesIds {
```

Comment:
Why do we need this interface? It can be a simple array of strings instead I think

Reply:
removed this interface